easybuild/easyblocks/generic/rubygem.py

```diff
@@ -100,6 +100,6 @@ def make_module_extra(self):
         """Extend $GEM_PATH in module file."""
         txt = super(RubyGem, self).make_module_extra()
         # for stand-alone Ruby gem installs, $GEM_PATH needs to be updated
-        if not self.is_extension:
+        if self.cfg.__dict__.get('software_name') != 'Ruby':
             txt += self.module_generator.prepend_paths('GEM_PATH', [''])
-        return txt
+        return super(RubyGem, self).make_module_extra(txt)
```

Comment:
Does this actually change anything? What's there should work...

Reply:
I think the original code was simply not executed. This change, indeed does not make any change, I just simply copied this part from the rpackage EasyBlock.

Reply:
Well, it's wrong as it is now, since we already call out to `super(RubyGem, self).make_module_extra()` when defining `txt`, see above.

It should be reinstated to just `return txt` imho.